src/Filtering/CMakeLists.txt

```diff
@@ -1,26 +1,34 @@
-add_subdirectory_if_module_enabled(BinaryMathematicalMorphology)
-add_subdirectory_if_module_enabled(ImageGrid)
-add_subdirectory_if_module_enabled(ImageFeature)
-add_subdirectory_if_module_enabled(FastMarching)
-add_subdirectory_if_module_enabled(Colormap)
-add_subdirectory_if_module_enabled(ImageIntensity)
-add_subdirectory_if_module_enabled(ImageFilterBase)
-add_subdirectory_if_module_enabled(Thresholding)
-add_subdirectory_if_module_enabled(QuadEdgeMeshFiltering)
-add_subdirectory_if_module_enabled(ImageCompare)
-add_subdirectory_if_module_enabled(MathematicalMorphology)
-add_subdirectory_if_module_enabled(Smoothing)
-# TODO: Need to re-enable ImageGraadient
-# add_subdirectory_if_module_enabled(ImageGradient)
-add_subdirectory_if_module_enabled(AnisotropicSmoothing)
-add_subdirectory_if_module_enabled(FFT)
-add_subdirectory_if_module_enabled(AntiAlias)
-add_subdirectory_if_module_enabled(LabelMap)
-add_subdirectory_if_module_enabled(ImageStatistics)
-add_subdirectory_if_module_enabled(ImageFusion)
-add_subdirectory_if_module_enabled(Convolution)
-add_subdirectory_if_module_enabled(DistanceMap)
-add_subdirectory_if_module_enabled(ImageCompose)
-add_subdirectory_if_module_enabled(ImageLabel)
-add_subdirectory_if_module_enabled(CurvatureFlow)
-add_subdirectory_if_module_enabled(Path)
+set(Filtering_MODULES
+  AnisotropicSmoothing
+  AntiAlias
+  BinaryMathematicalMorphology
+  Colormap
+  Convolution
+  CurvatureFlow
+  DistanceMap
+  FastMarching
+  FFT
+  ImageCompare
+  ImageCompose
+  ImageFeature
+  ImageFilterBase
+  ImageFusion
+  ImageGradient
+  ImageGrid
+  ImageIntensity
+  ImageLabel
+  ImageStatistics
+  LabelMap
+  MathematicalMorphology
+  "Path"
```

Comment:
Why is this quoted?

Reply:
Quotes are to distinguish to the reader that we are using "Path" as a module name rather than as the CMake `PATH` keyword or the system `PATH` variable. Without the quotes my IDE highlights `Path` as a keyword. Would it be less confusing to remove the quotes?

Reply:
Yes, it would be helpful to remove the quotes or add a comment explaining why they are there.

Reply:
Unquoted